sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobCopyFromUrlOptions.java

```diff
@@ -118,4 +120,20 @@ public BlobCopyFromUrlOptions setDestinationRequestConditions(BlobRequestConditi
         this.destinationRequestConditions = destinationRequestConditions;
         return this;
     }
+
+    /**
+     * @return auth header for access to source.
+     */
+    public HttpAuthorization getSourceAuthorization() {
+        return sourceAuthorization;
+    }
+
+    /**
+     * @param sourceAuthorization auth header for access to source.
```

Comment:
we should document that we take only Bearer token at this time here and places like this.